lib/services/nativescript-project-plugins-service.ts

```diff
@@ -48,7 +49,16 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 	public findPlugins(keywords: string[]): IFuture<IBasicPluginInformation[]> {
 		return ((): IBasicPluginInformation[] => {
 			let findPluginByNameFutures = _.map(keywords, keyword => this.findPluginsByName(keyword));
-			return getFuturesResults<IBasicPluginInformation>(findPluginByNameFutures, plugins => !!plugins && !!plugins.length);
+			Future.wait(findPluginByNameFutures);
+			let findPluginsResults: IBasicPluginInformation[][] = findPluginByNameFutures.map(f => f.get()).filter(plugins => !!plugins && !!plugins.length);
+
+			let pluginsNamesInGroups = findPluginsResults.map(res => _.map(res, r => r.name));
+			let matchingNames = _.intersection.apply(this, pluginsNamesInGroups);
+
+			return _(findPluginsResults)
+					.flatten<IBasicPluginInformation>()
+					.filter(basicInfo => _.contains(matchingNames, basicInfo.name))
```

Comment:
this way each plugin, that is found in both collections, will be shown twice, show only one instance of a plugin